src/soundmanager.h

```diff
@@ -62,13 +62,13 @@ class SoundManager : public QObject {
     // open, this method simply runs through the list of all known soundcards
     // (from PortAudio) and attempts to close them all. Closing a soundcard that
     // isn't open is safe.
-    void closeDevices();
+    void closeDevices(bool wait);
```

Comment:
I also noticed that func(false) / func(true) is hard to read and thought about introducing an enum. But this might be overkill if it is only used for this special purpose.

On the other hand I don't think that is is a good idea to let the caller decide on how long to wait. This is an implementation detail of the function and should not be exposed unless it depends on circumstances that only the caller might be aware of.